provisioning/packer-virtual-machines/deploy/job.yaml

```diff
@@ -0,0 +1,60 @@
+apiVersion: batch/v1
```

Comment:
Seems to me this is identical to the previous manifest, but for the `HAS_DESKTOP` variable. Would remove, and add the variable to the other. The more code/manifests are duplicated, the more inconsistencies you get.